CHANGELOG.md

```diff
@@ -9,7 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
--   Consuming codes on a device different from the
+-   Consuming codes on a device different from the one that created it
```

Comment:
Might wanna give more details about this - that we show a button that the user has to click and it's done cause of XYZ reason.